backend/app/Http/Controllers/UserController.php

```diff
@@ -14,8 +14,15 @@ public function register(Request $request){
 			'name'=> 'required|string|max:255',
 			'email'=>'required|string|email|max:255|unique:users',
 			'password'=>'required|min:6',
+			'confirmPassword'=> 'required|string',
```

Comment:
```suggestion
			'password'=>'required|confirmed|min:6',
```
you can check the confirmed password like this